deploy/packages/rpm/Makefile

```diff
@@ -36,6 +38,8 @@ all: | $(BUILT)
 		--define "_post_addition $(POST_ADDITION)" \
 		--define "_preun_addition $(PREUN_ADDITION)" \
 		--define "_sharedstatedir /var/lib" \
+		--defined "_service_src $(SERVICE_SRC)"
```

Comment:
I don’t think these macros are used in emqx.spec
Maybe the build environment doesn’t have systemd installed which failed the check at line 19

Reply:
I add `_service_src/_service_dst` in emqx.spec now.
Building package without systems is ok, it only copy emqx.services into package.
but install package without emqx.services,  will make yum install success, but systemctl is broken.